### DISCUSSION TITLE
Merge blocked even though deploy is successful

### DISCUSSION THREAD

**Discussion by alexboii** [vercel/community#3657] [Help]
### Summary

We require deployments to succeed before merging to "main" in our repo. The deployment in question is called "Preview - web-deploy." As you can see in the screenshot, the deployment passes, but for some reason we are still blocked from merging our PRs

![image](https://github.com/vercel/community/assets/10774468/a32578e1-a05a-4c29-a3c3-c4e30aebc041)

### Steps to Reproduce

- Enable "require deployments to succeed before merging" option with "Preview - web-deploy" environment on main branch
- Open a PR & wait for deployments to complete 
- See "Merge Blocked" error

**Comment by alexboii** [comment]
Still unanswered :)

**Comment by amyegan** [comment]
This appears to be a problem with project configuration on GitHub. Please review the repo's merge settings. 🙏

**Comment by jackfrosty908** [comment]
@alexboii did you ever find out the root cause for this?
